src/icons/Lock.js

```diff
@@ -0,0 +1,12 @@
+// @flow
+
+import React from 'react'
+
+export default ({ size, ...p }: { size: number }) => (
+  <svg width={size} height={size} {...p}>
+    <path
```

Comment:
general practice would be either to do a PureComponent or to at least move `<path>` into an outer const, so we don't render again & again. (this might help a bit react perf if we do that for all icons.)